src/server/generic_family_test.cc

```diff
@@ -478,7 +478,13 @@ TEST_F(GenericFamilyTest, Sort) {
 
   Run({"set", "foo", "bar"});
   ASSERT_THAT(Run({"sort", "foo"}), ErrArg("WRONGTYPE "));
-  ;
+
+  Run({"rpush", "list-3", ""});
+  ASSERT_THAT(Run({"sort", "list-3"}), "");
+
+  Run({"rpush", "list-3", "2", "0", "", "-0.14", "0.12", "-0", "-123123", "7654"});
+  ASSERT_THAT(Run({"sort", "list-3"}).GetVec(),
+              ElementsAre("-123123", "-0.14", "", "", "-0", "0", "0.12", "2", "7654"));
```

Comment:
`-0` how can `zero` have `sign` :scream: Abseil parses this ?!

Reply:
double type support 0 and -0. it's different numbers

Reply:
Uh the sign bit on IEEE :face_exhaling: